File: coding_meetup_4.js
function getFirstPython(list) {
    // Thank you for checking out my kata :)
    let firstPythonDev=list.find(dev=>{
     return dev.language==='Python'
    })
    console.log(firstPythonDev)
    if(firstPythonDev===undefined){
      return "There will be no Python developers"
    }else{
      return `${firstPythonDev.firstName}, ${firstPythonDev.country}`
    }
  }

File: coding_meetup_4.py
def get_first_python(users):
    python_devs = list(filter(lambda x: x["language"] == "Python", users))
    if not python_devs:
        return "There will be no Python developers"
    return f"{python_devs[0]['first_name']}, {python_devs[0]['country']}"